### PR TITLE
Docs OpenSSH tsh instructions

### PR DESCRIPTION
**Pull request by stevenGravy**
Provide guidance on connecting to a OpenSSh with TSH

### CODE REVIEW

File: docs/4.2/admin-guide.md [hunk 1]
```diff
@@ -1956,6 +1956,16 @@ To allow access for all users:
   + Copy `teleport-user-ca.pub` to `/etc/ssh/teleport-user-ca.pub`
   + Update `sshd` configuration (usually `/etc/ssh/sshd_config` ) to point to
     this file: `TrustedUserCAKeys /etc/ssh/teleport-user-ca.pub`
+ 
+ To connect to the OpenSSH server via `tsh` add the `--port=<ssh port>` with the `tsh ssh` command:
```
Comment (webvictim) [inline docs/4.2/admin-guide.md:1960]: ...via `tsh`, add `--port=<ssh port>` to the `tsh ssh` command:

(add comma, fix formatting)

File: docs/4.2/admin-guide.md [hunk 2]
```diff
@@ -1956,6 +1956,16 @@ To allow access for all users:
...
+  + Example ssh to `database.work.example.com` as `root` with a OpenSSH server on port 22 via `tsh`
```
Comment (webvictim) [inline docs/4.2/admin-guide.md:1961]: ...via `tsh`:

(add colon)

File: docs/4.2/admin-guide.md [hunk 3]
```diff
@@ -1956,6 +1956,16 @@ To allow access for all users:
...
+ ```bash
+    tsh ssh --port=22 root@database.work.example.com
+ ```
+    
+!!! warning "Warning"
+
+    The user being connected as must be available in the Teleport user configuration or role.
```
Comment (webvictim) [inline docs/4.2/admin-guide.md:1968]: The principal (username) being used to connect must be listed in the Teleport user/role configuration.

(bit more readable, plus uses the term 'principal' which people could potentially Ctrl+F for if they're getting the "name is not a listed principal" error)

**Review by benarent — CHANGES_REQUESTED** [review]
Can you also add this into 4.3, if you update from master those documents are available now.

**Pull request closed without merging**